domain/src/main/java/com/bottlerocketstudios/brarchitecture/domain/models/Repository.kt

```diff
@@ -0,0 +1,13 @@
+package com.bottlerocketstudios.brarchitecture.domain.models
+
+import java.time.ZonedDateTime
+
+data class Repository(
+    val scm: String?,
+    val name: String?,
+    val owner: User?,
+    val workspace: Workspace?,
+    val isPrivate: Boolean?,
+    val description: String?,
+    val updated: ZonedDateTime?
```

Comment:
@VanyaHuamanBottleRocket nit: Add trailing comma for less noisy future diffs :)

```suggestion
    val updated: ZonedDateTime?,
```

Reply:
Updating